test/mailers/invitation_mailer_test.rb

```diff
@@ -7,23 +7,23 @@ class InvitationMailerTest < ActionMailer::TestCase
   include Rails.application.routes.url_helpers
   def mail_url(route, options = nil)
     url_for([
-      route,
-      Rails.application.config.action_mailer.default_url_options.merge(options)
-    ])
+              route,
+              Rails.application.config.action_mailer.default_url_options.merge(options)
+            ])
```

Comment:
Hmmm, I feel like I want to set [`Layout/IndentFirstArrayElement` to `EnforcedStyle: consistent`](https://rubocop.readthedocs.io/en/latest/cops_layout/#layoutindentfirstarrayelement). Thoughts? (Not worth changing if you don’t agree; my opinion here is not a strong one.)

Reply:
I'm a big fan of `consistent` over `indentFirst` in every cop I can think of. but I don't feel strongly and the `--safe-auto-correct` flag makes the cost of change very low.

Once this PR is merged, then let's do a second pass on any style changes.